src/core/qgslegendsettings.h

```diff
@@ -178,6 +178,18 @@ class CORE_EXPORT QgsLegendSettings
      */
     void setMapScale( double scale ) { mMapScale = scale; }
 
+    /**
+     * Returns the factor of map units per pixel for symbols with size given in map units calculated by mDpi and mMmPerMapUnit
```

Comment:
Better don't use private symbols in public API docs.

I.e. just write "dpi" and "mmPerMapUnit".